flag/service/service.go

```diff
@@ -0,0 +1,10 @@
+package service
+
+import (
+	"github.com/giantswarm/operatorkit/flag/service/kubernetes"
```

Comment:
Looks like this is missing?

Reply:
I don't think so - it's from operatorkit, right?